example/retarget.cproject.yml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Open-CMSIS-Pack/devtools/schemas/projmgr/1.5.0/tools/projmgr/schemas/cproject.schema.json
```

Comment:
It was suggested that we would no longer maintain the `# yaml-language-server: ...` line in our project files.